hello/README.md

```diff
@@ -116,7 +116,7 @@ To try out the staff's implementation of this problem, execute
 ./hello
 </pre>
 
-within <a href="https://sandbox.cs50.io/0c8ebcaa-ca6e-45fa-bff2-043e5e6d83ec">this sandbox</a>.
+within <a href="http://bit.ly/2Qp0a2g">this sandbox</a>.
```

Comment:
How come this is not a markdown link like the others?

Reply:
Can we change this to Markdown, @curiouskiwi ?